CloudNotes/CloudNotes/View/ListViewController.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  ListTableViewController.swift
+//  CloudNotes
+//
+//  Created by Ellen on 2021/09/03.
+//
+
+import UIKit
+
+class ListViewController: UIViewController {
+    private var tableView: UITableView!
+    private var memo: [MemoList] = []
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        prepareMemo()
+        setUp()
+    }
+    
+    private func setUp() {
+        tableView = UITableView()
+        view.addSubview(tableView)
+        title = "메모"
+        tableView.dataSource = self
+        tableView.register(ListTableViewCell.classForCoder(), forCellReuseIdentifier: ListTableViewCell.cellIdentifier)
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        tableView.rowHeight = (UIScreen.main.bounds.height) / 12
+        NSLayoutConstraint.activate([
+                                        tableView.topAnchor.constraint(equalTo: view.topAnchor),
+                                        tableView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+                                        tableView.leftAnchor.constraint(equalTo: view.leftAnchor),
+                                        tableView.rightAnchor.constraint(equalTo: view.rightAnchor)])
+    }
+    
+    private func prepareMemo() {
+        let jsonHandler = JSONHandler()
+        guard let jsonData = JsonUtil.loadJsonData("sample") else { return }
+        guard let data = jsonHandler.decode(with: jsonData, to: [MemoList].self) else { return }
```

Comment:
guard문을 사용하셨네요!🙌
우선 두 guard문의 else는 동일하게 `return`을 호출해주고 있는데요.
```suggestion
        guard let jsonData = JsonUtil.loadJsonData("sample"),
              let data = jsonHandler.decode(with: jsonData, to: [MemoList].self)  else { return }
 }
```
이런식으로 분리하지 않고 사용하는건 어떤가요?